Documentation/Blazorise.Docs/Program.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+
+namespace Blazorise.Docs;
+
+public class Program
+{
+    public static async Task Main( string[] args )
+    {
+        var builder = WebAssemblyHostBuilder.CreateDefault( args );
+
+        await builder.Build().RunAsync();
+    }
```

Comment:
Oh, I see what you mean with being a client project... you setup an actual wasm project.. that's strange does the feature not work without explicitly making it wasm by default?

Reply:
Because I don't understand how this entry point would have any effect in the actual entry point we use **Blazorise.Docs.Server** to setup everything.

Reply:
See this comment

![image](https://github.com/Megabit/Blazorise/assets/900302/c2434e37-8514-4dd4-adbb-0b2aad7c363b)

Reply:
That's true for wasm projects, But you're not bootstraping a pure WASM project. You're going through a server ultimately which knows of what needs bootstraping.

Reply:
But maybe it's just my lack of knowledge in the new net8 changes. If you've tried it and it did not work, then ok.

PS: Your image shows difference between sdk.wasm and sdk.server.
You changed from sdk.razor to sdk.wasm. I still think sdk.razor is enough for the purpose of it being just a component library... but if you researched the feature and you found out that you absolutely need the sdk.wasm then ok.